src/components/Dropdown.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { Dispatch, ReactElement, SetStateAction, useState } from 'react'
+import DropDownPicker, { ItemType, ValueType } from 'react-native-dropdown-picker'
+import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
+import styled, { useTheme } from 'styled-components/native'
+
+import { Color } from '../constants/theme/colors'
+import { ContentError } from './text/Content'
+
+const StyledDropDownPicker = styled(DropDownPicker)<{ borderColor: string }>`
+  border-radius: 0;
+  height: 60px;
+  border: 1px solid ${props => props.borderColor};
+`
+
+const ErrorContainer = styled.View`
+  margin-top: ${props => props.theme.spacings.xs}
+  min-height: ${props => props.theme.spacings.lg};
+`
+
+interface DropdownProps<T> {
+  placeholder: string
+  items: ItemType<string | number>[]
+  itemKey: string
+  multiSelect?: boolean
+  setValue: Dispatch<SetStateAction<T | null>>
+  value: T | null
+  schema?: { label: string; value: string }
```

Comment:
you can map your items array with the schema of the dropdown. but probably you are right to keep a default schema since we don't need more than label and value